tools/wheels/cibw_before_build.sh

```diff
@@ -17,6 +16,14 @@ if [[ $RUNNER_OS == "Linux" || $RUNNER_OS == "macOS" ]] ; then
     basedir=$(python tools/openblas_support.py)
     cp -r $basedir/lib/* /usr/local/lib
     cp $basedir/include/* /usr/local/include
+    # TODO: don't copy directories if not arm64,
+    # but we need a way to know if cibuildwheel is cross-compiling arm64 to do this
```

Comment:
I noticed that builds for both architectures are done in a single build matrix entry. Not sure if that is how it must be for cibuildwheel to be happy - otherwise, `os: macos-latest` (which is a macOS 10.15 image) can be used for `x86_64` and `os: macos-11` for `arm64`.

Similarly, `MACOSX_DEPLOYMENT_TARGET` can be used - it's `10.13` (I think) as set by cibuildwheel for `x86_64`, and it must be `11` for `arm64`.

Reply:
I think I figured out a better way to detect arm64, but unfortunately I think cibuildwheel only supports doing the cross-compiling thing. I remember that if I put in ``macosx_arm64`` or ``macosx_universal2`` for the ``platform`` variable, cibuildwheel errored(presumably because there are no macos arm64 CI providers).